rest-api-spec/src/main/resources/rest-api-spec/api/open_reader.json

```diff
@@ -0,0 +1,65 @@
+{
+  "open_reader":{
+    "documentation":{
+      "url":"https://www.elastic.co/guide/en/elasticsearch/reference/master/reader-context.html",
+      "description":"Open and keep point-in-time readers that can be used in subsequent search requests"
+    },
+    "stability":"beta",
+    "url":{
+      "paths":[
+        {
+          "path":"/_open_reader",
+          "methods":[
+            "POST"
+          ]
+        },
+        {
+          "path":"/{index}/_open_reader",
+          "methods":[
+            "POST"
+          ],
+          "parts":{
+            "index":{
+              "type":"list",
+              "description":"A comma-separated list of index names to search; use `_all` or empty string to perform the operation on all indices"
+            }
+          }
+        }
+      ]
+    },
+    "params":{
+      "preference":{
+        "type":"string",
+        "description":"Specify the node or shard the operation should be performed on (default: random)"
+      },
+      "routing":{
+        "type":"string",
+        "description":"Specific routing value"
+      },
+      "ignore_unavailable":{
+        "type":"boolean",
+        "description":"Whether specified concrete indices should be ignored when unavailable (missing or closed)"
+      },
+      "allow_no_indices":{
```

Comment:
I wonder if we should always throw an error if there is no concrete indices that match ?

Reply:
Removed in https://github.com/elastic/elasticsearch/pull/55265/commits/51ddf7759d11214fa602a84290414a5f5f0b3233